scripts/airgapped/deploy-1.8.sh

```diff
@@ -0,0 +1,168 @@
+# This script assumes the presence of `retagged-images.txt` file and `charms` directory in the root dir.
+# You can generate these files using the [Airgap utility scripts](https://github.com/canonical/bundle-kubeflow/blob/main/scripts/airgapped/README.md). 
+
+IMAGES=~/retagged-images.txt
+CHARMS_DIR=~/charms
```

Comment:
```suggestion
# Define paths and variables
IMAGES="$HOME/retagged-images.txt"
CHARMS_DIR="$HOME/charms"
```

I believe using $HOME rather then ~ is best practice ([Here](https://stackoverflow.com/questions/5930671/why-use-home-over-tilde-in-a-shell-script) is some unrelated example why)